mahotas/tests/LineDetectorByHough.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2008-2014, Jasson <1917098992@qq.com>
+# License: MIT (see COPYING file)
+
+import math
+import numpy as np
+
+class Line():
+    def __init__(self,r,angle):
+        self.r = r
+        self.angle = angle
+    def __eq__(self, other):
+        if self.r == other.r and self.angle == other.angle:
+            return True
+        else:
+            return False
+    def __hash__(self):
+        result = 31 + int(self.r)
+        result = 31 * result + int(self.angle * 100)
+        #print result
+        return result
+
+class Counter(dict):
+    def __missing__(self, key):
+        return 0
+
+class LineDetectorByHough:
+    countersDICT = Counter(dict())
```

Comment:
This makes the `countsDICT` be a class-level variable, you must assign to it inside the `__init__` constructor

Reply:
I try to do it, but it does not work well - unit test failed